### PR TITLE
add reMarkable.app

### PR DESCRIPTION
**Pull request by bernadinm**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [X] `brew audit --cask {{cask_file}}` is error-free.
- [X] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask {{cask_file}}` worked successfully.
- [X] `brew install --cask {{cask_file}}` worked successfully.
- [X] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by bernadinm** [comment]
The build fails because its lookign for the `appcast` but there isn't one for this app so it's omitted.

```bash
"$(brew --repository)/Library/Taps/homebrew/homebrew-cask/developer/bin/find-appcast" '/Applications/reMarkable.app/'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/universal-darwin20/rbconfig.rb:229: warning: Insecure world writable dir /Users/usernamehere in PATH, mode 040777
Looking for Sparkle appcast… Not found.
Looking for electron-builder appcast… Not found.
```

**Comment by bernadinm** [comment]
The instructions test works locally:
```bash
brew audit --new-cask remarkable
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/universal-darwin20/rbconfig.rb:229: warning: Insecure world writable dir /Users/usernamehere in PATH, mode 040777
audit for remarkable: passed
```

**Comment by vitorgalvao** [comment]
> The build fails because it's looking for the `appcast`

No, it’s failing because there’s no [`livecheck`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/livecheck.md). It needs that and a [`zap`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md) (see [`homebrew-createzap`](https://github.com/nrlquaker/homebrew-createzap) for an external project which can help).

File: Casks/remarkable.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "remarkable" do
+  version "2.5.0"
+  sha256 "ec6242a123c33af8a1d4d6d476b54889698d9e5d3bff73551ae9bb7eacb3439f"
+
+  url "https://downloads.remarkable.com/desktop/production/mac/reMarkable-#{version}.dmg"
+  name "ReMarkable"
+  desc "Desktop application enables you to view, organize, import, and download files"
```
Comment (vitorgalvao) [inline Casks/remarkable.rb:7]: This doesn’t explain what the app does or lets you do. Presumably to does those things in interaction with the tablet; that needs to be explained.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by miccal** [comment]
Closing die to lack of reply.

**Pull request closed without merging**